### DISCUSSION TITLE
Could be false flags, but lots of trojan warnings.

### DISCUSSION THREAD

**Discussion by bonecast** [zanllp/infinite-image-browsing#611] [General]
from the latest release .exe

https://www.virustotal.com/gui/file/4aaab3f072ea570bfa97e0d62a6a6758dfa267b30bf4104c5441026c1998f1e6

https://virusscan.jotti.org/en-US/filescanjob/zp69y33j6l

**Comment by zanllp** [comment]
This false positive is due to Nuitka, a tool that compiles Python scripts into exe files. If this issue concerns you, you can compile it yourself or run it directly with Python. All of IIB's code uses open-source libraries that are popular in the industry. I currently do not have the spare capacity to address this false positive.

https://github.com/zanllp/sd-webui-infinite-image-browsing/issues/490

https://github.com/zanllp/sd-webui-infinite-image-browsing/issues/397

<img width="737" alt="image" src="https://github.com/zanllp/sd-webui-infinite-image-browsing/assets/25872019/0a7e84a3-d317-494b-b525-21c1d3dc219b">

**Comment by zanllp** [comment]
In the latest release, a PyInstaller-compiled version has been added, significantly reducing the likelihood of false positive virus detections. https://github.com/zanllp/sd-webui-infinite-image-browsing/releases/tag/v0.17.0
